### PR TITLE
[RIP-9]Add the operation part

### PR DESCRIPTION
**Pull request by chuenfaiy**
## What is the purpose of the change

RocketMQ Developer Guide doc

## Brief changelog

Add the file named `operation.md` of chapter 8

## Verifying this change

no functional change

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/cn/operation.md [hunk 1]
```diff
@@ -0,0 +1,1408 @@
...
+### 1   集群搭建
+
+#### 1.1 单Master模式
+
+这种方式风险较大，一旦Broker重启或者宕机时，会导致整个服务不可用。不建议线上环境使用,可以用于本地测试。
+
+##### 1）启动 NameServer
+
+```bash
+### 首先启动Name Server，例如机器IP为：192.161.1:9876
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: “例如机器IP为：192.161.1:9876” 与下文无关可否去掉

File: docs/cn/operation.md [hunk 2]
```diff
@@ -0,0 +1,1408 @@
...
+  <td height=23 class=xl63 width=177 style='height:17.0pt;width:133pt'>名称</td>
+  <td class=xl64 width=175 style='width:131pt'>含义</td>
+  <td class=xl64 width=177 style='width:133pt'>命令选项</td>
+  <td class=xl64 width=185 style='width:139pt'>说明</td>
+ </tr>
+ <tr height=132 style='height:99.0pt'>
+  <td rowspan=8 height=593 class=xl68 width=163 style='border-bottom:1.0pt;
+  height:444.0pt;border-top:none;width:122pt'>updateTopic</td>
+  <td rowspan=8 class=xl70 width=135 style='border-bottom:1.0pt;
+  border-top:none;width:101pt'>创建Topic或更新Topic配置</td>
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: 建议改为“创建/更新Topic配置“

File: docs/cn/operation.md [hunk 3]
```diff
@@ -0,0 +1,1408 @@
...
+  <td class=xl66 width=159 style='width:119pt'>cluster 名称，表示 topic 所在集群（集群可通过
+  clusterList 查询）</td>
+ </tr>
+ <tr height=23 style='height:17.0pt'>
+  <td height=23 class=xl65 width=149 style='height:17.0pt;width:112pt'>-h-</td>
+  <td class=xl66 width=159 style='width:119pt'>打印帮助</td>
+ </tr>
+ <tr height=57 style='height:43.0pt'>
+  <td height=57 class=xl65 width=149 style='height:43.0pt;width:112pt'>-n</td>
+  <td class=xl66 width=159 style='width:119pt'>nameserve 服务地址，格式 ip:port</td>
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: nameserve?

File: docs/cn/operation.md [hunk 4]
```diff
@@ -0,0 +1,1408 @@
...
+ </tr>
+ <tr height=76 style='height:57.0pt'>
+  <td height=76 class=xl65 width=149 style='height:57.0pt;width:112pt'>-p</td>
+  <td class=xl66 width=159 style='width:119pt'>指定新 topic 的权限限制( W=2|R=4|WR=6 )</td>
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: delete ”限制"

File: docs/cn/operation.md [hunk 5]
```diff
@@ -0,0 +1,1408 @@
...
+  <td class=xl66 width=159 style='width:119pt'>cluster 名称，表示删除某集群下的某个 topic （集群
+  可通过 clusterList 查询）</td>
+ </tr>
+ <tr height=23 style='height:17.0pt'>
+  <td height=23 class=xl65 width=149 style='height:17.0pt;width:112pt'>-h</td>
+  <td class=xl66 width=159 style='width:119pt'>打印帮助</td>
+ </tr>
+ <tr height=57 style='height:43.0pt'>
+  <td height=57 class=xl65 width=149 style='height:43.0pt;width:112pt'>-n</td>
+  <td class=xl66 width=159 style='width:119pt'>nameserve 服务地址，格式 ip:port</td>
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: nameserve?

File: docs/cn/operation.md [hunk 6]
```diff
@@ -0,0 +1,1408 @@
...
+  <td height=39 class=xl65 width=177 style='height:29.0pt;width:133pt'>-i</td>
+  <td class=xl66 width=185 style='width:139pt'>打印间隔，单位秒</td>
+ </tr>
+ <tr height=95 style='height:71.0pt'>
+  <td rowspan=8 height=391 class=xl67 width=177 style='border-bottom:1.0pt;
+  height:292.0pt;border-top:none;width:133pt'>clusterRT</td>
+  <td rowspan=8 class=xl70 width=175 style='border-bottom:1.0pt;
+  border-top:none;width:131pt'>发送消息检测集群各Broker RT。消息发往${BrokerName} Topic。</td>
+  <td class=xl65 width=177 style='width:133pt'>-a</td>
+  <td class=xl66 width=185 style='width:139pt'>amount，每次探测的探测总数，RT = 总时间 /
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: “每次探测的总数”

File: docs/cn/operation.md [hunk 7]
```diff
@@ -0,0 +1,1408 @@
...
+  <td height=23 class=xl67 width=87 style='height:17.0pt;width:65pt'>-h</td>
+  <td class=xl68 width=87 style='width:65pt'>打印帮助</td>
+ </tr>
+ <tr height=57 style='height:43.0pt'>
+  <td height=57 class=xl67 width=87 style='height:43.0pt;width:65pt'>-n</td>
+  <td class=xl68 width=87 style='width:65pt'>nameserve 服务地址，格式 ip:port</td>
+ </tr>
+ <tr height=57 style='height:43.0pt'>
+  <td rowspan=6 height=256 class=xl69 width=191 style='border-bottom:1.0pt;
+  height:192.0pt;border-top:none;width:143pt'>BrokerConsumeStats</td>
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: brokerConsumeStats

File: docs/cn/operation.md [hunk 8]
```diff
@@ -0,0 +1,1408 @@
...
+  <td height=23 class=xl67 width=87 style='height:17.0pt;width:65pt'>-h</td>
+  <td class=xl68 width=87 style='width:65pt'>打印帮助</td>
+ </tr>
+ <tr height=57 style='height:43.0pt'>
+  <td height=57 class=xl67 width=87 style='height:43.0pt;width:65pt'>-n</td>
+  <td class=xl68 width=87 style='width:65pt'>nameserve 服务地址，格式 ip:port</td>
+ </tr>
+ <tr height=57 style='height:43.0pt'>
+  <td rowspan=3 height=137 class=xl69 width=191 style='border-bottom:1.0pt;
+  height:103.0pt;border-top:none;width:143pt'>BrokerStatus</td>
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: brokerStatus

File: docs/cn/operation.md [hunk 9]
```diff
@@ -0,0 +1,1408 @@
...
+  <td height=57 class=xl67 width=87 style='height:43.0pt;width:65pt'>-n</td>
+  <td class=xl68 width=87 style='width:65pt'>nameserve 服务地址，格式 ip:port</td>
+ </tr>
+ <tr height=225 style='height:169.0pt'>
+  <td rowspan=6 height=390 class=xl69 width=87 style='border-bottom:1.0pt;
+  height:292.0pt;border-top:none;width:65pt'>queryMsgByOffset</td>
+  <td rowspan=6 class=xl72 width=87 style='border-bottom:1.0pt;
+  border-top:none;width:65pt'>根据 Offset 查询消息</td>
+  <td class=xl67 width=87 style='width:65pt'>-b</td>
+  <td class=xl68 width=87 style='width:65pt'>Broker 名称，表示订阅组建在该 Broker（这里需要注意
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: "表示订阅组建在该"?

File: docs/cn/operation.md [hunk 10]
```diff
@@ -0,0 +1,1408 @@
...
+
+在某些情况下，Consumer需要将消费位点重置到1-2天前，这时在内存有限的Master Broker上，CommitLog会承载比较重的IO压力，影响到该Broker的其它消息的读与写。可以开启`slaveReadEnable=true`，当Master Broker发现Consumer的消费位点与CommitLog的最新值的差值的容量超过该机器内存的百分比（`accessMessageInMemoryMaxRatio=40%`），会推荐Consumer从Slave Broker中去读取数据，降低Master Broker的IO。
+
+#### 3.5 性能调优问题
+
+异步刷盘建议使用自旋锁，同步刷盘建议使用重入锁，调整Broker配置项`useReentrantLockWhenPutMessage`，默认为false；异步刷盘建议开启`TransientStorePoolEnable`；建议关闭transferMsgByHeap，提高拉消息效率；同步刷盘建议适当增大`sendMessageThreadPoolNums`，具体配置需要经过压测。
+
+#### 3.6 在RocketMQ中msgId和offsetMsgId的含义与区别
+
+我们使用RocketMQ完成生产者客户端消息发送后，通常会看到如下日志打印信息：
```
Comment (wlliqipeng) [inline docs/cn/operation.md]: “我们”delete

**Comment by chuenfaiy** [comment]
@wlliqipeng All Error Is Correct!

**Pull request merged**